README.md

```diff
@@ -176,9 +186,15 @@ And define our draw() function:
             relationships: {
                 INTERACTS: {
                     value: "weight"
+                    [NeoVis.NEOVIS_ADVANCED_CONFIG]: {
+                        function: {
+                            title: (relationship) => relationship.properties.weight,
```

Comment:
Also here you are using title wrong 😅

Reply:
How would it be?

Reply:
Same as the title function in nodes 😅

Reply:
https://github.com/neo4j-contrib/neovis.js/pull/346#discussion_r1197142214
I already commented here, my bad for not commenting on both lines 😅